lib/sentry/config.ex

```diff
@@ -14,6 +14,11 @@ defmodule Sentry.Config do
   @default_context_lines 3
   @default_sample_rate 1.0
 
+  @permitted_log_level_atom_values ~w(debug info warn error)a
+  @permitted_log_level_string_values ~w(debug info warn error)
```

Comment:
The string values can be removed, and the atom set can be renamed to `permitted_log_level_values`.  Though Sentry does a lot of it currently, my goal is to limit how much the library transforms things from the original config.  Especially in cases like this where we can check the validity of the config early, I think it is okay to be stricter.

Reply:
Done.